Tests/Unit/Encodings/PaymentRequestTests.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import LibMobileCoin
+@testable import MobileCoin
+import XCTest
+
+class PaymentRequestTests: XCTestCase {
+
+    static let publicAddress = (try? PublicAddress.Fixtures.Init())!.accountKey.publicAddress
+    static let externalPublicAddress = External_PublicAddress(publicAddress)
+    static let value: UInt64 = 123
+    static let memo = "test memo"
+
+    func testEncodingToPrintableNoValueNoMemom() throws {
+        XCTAssertNoThrow(evaluating: {
```

Comment:
I think the `XCTAssertNoThrow` is unnecessary. If any line in the test throws it will be considered a failure by XCTest.